package/base-files/Makefile

```diff
@@ -227,9 +227,18 @@ endif
 	$(if $(CONFIG_TARGET_PREINIT_DISABLE_FAILSAFE), \
 		rm -f $(1)/etc/banner.failsafe,)
 
+ifdef CONFIG_USE_APK
 	mkdir -p $(1)/etc/apk/
-	$(call FeedSourcesAppend,$(1)/etc/apk/repositories)
+	$(call FeedSourcesAppendAPK,$(1)/etc/apk/repositories)
 	$(VERSION_SED_SCRIPT) $(1)/etc/apk/repositories
+else
+	$(if $(CONFIG_CLEAN_IPKG),, \
```

Comment:
We probably want a similar option also for APK, allowing the removal of package list and package manager state from the final image.